modules/backend/widgets/form/partials/_field_switch.htm

```diff
@@ -2,7 +2,7 @@
 <div class="field-switch">
     <label><?= e(trans($field->label)) ?></label>
     <?php if ($field->comment): ?>
-        <p class="help-block"><?= $field->commentHtml ? trans($field->comment) : e(trans($field->comment)) ?></p>
+        <p class="help-block"><?= e(trans($field->comment)) ?></p>
```

Comment:
Why did you change this?

Reply:
Honestly I don't remembered to change that line of code, maybe that's because I have another project where I have the changes and I copy the entire file, but the one that's in the official octobercms develop branch is some or few commits ahead.

That line of code is not supossed to be changed :(

Reply:
Just change it back then :)